manifests/init.pp

```diff
@@ -46,10 +50,10 @@
   file { '/etc/drbd.d':
     ensure  => directory,
     mode    => '0644',
-    purge   => true,
+    purge   => false, 
```

Comment:
Not an expert on the module, but I think this purging is intentional and probably shouldn't be changed. If anything, it should be a parameter.